sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -60,7 +65,7 @@ export class CertificateClient {
     deleteContacts(options?: DeleteContactsOptions): Promise<CertificateContact[] | undefined>;
     deleteIssuer(issuerName: string, options?: DeleteIssuerOptions): Promise<CertificateIssuer>;
     getCertificate(certificateName: string, options?: GetCertificateOptions): Promise<KeyVaultCertificateWithPolicy>;
-    getCertificateOperation(certificateName: string, options?: GetCertificateOperationOptions): Promise<PollerLike<PollOperationState<CertificateOperation>, CertificateOperation>>;
+    getCertificateOperation(certificateName: string, options?: GetCertificateOperationOptions): Promise<PollerLikeWithPublicState<CertificateOperationPublicState, CertificateOperationPollOperationState, KeyVaultCertificateWithPolicy>>;
```

Comment:
thoughts @bterlson? I feel like we're definitely in "type too long" territory, imho

Reply:
Is there another approach? I couldn't think of one. Low creativity day.

Reply:
Alias the type inside the promise, e.g.

```ts
type GetCertificatePoller = PollerLikeWithPublicState<CertificateOperationPublicState, CertificateOperationPollOperationState, KeyVaultCertificateWithPolicy>;

getCertificateOperation(certificateName: string, options?: GetCertificateOperationOptions): Promise<GetCertificatePoller>;
```

Reply:
Doing it in a bit! Thank you!

Reply:
Pushed a fix! 🌞